src/components/QueryBuilder.vue

```diff
@@ -25,7 +25,8 @@
 					v-model="textInputValue"
 					:error="fieldErrors.value ?
 						{message: $i18n(fieldErrors.value.message), type: fieldErrors.value.type}: null"
-					:placeholder="$i18n('query-builder-input-value-placeholder')" />
+					:placeholder="$i18n('query-builder-input-value-placeholder')"
+					:disabled="selectedPropertyValueRelation === propertyValueRelation.Regardless" />
```

Comment:
We probably should put the `/>` on its own line. Then we don't have those extra line changes when we have to add another attribute.